src/init.cpp

```diff
@@ -191,11 +191,11 @@ static void CreateNewConfigFile()
     myConfig
         << "addnode=addnode-us-central.cycy.me\n"
         << "addnode=ec2-3-81-39-58.compute-1.amazonaws.com\n"
-        << "addnode=gridcoin.crypto.fans\n"
+        << "addnode=grcnode.thefoxie.eu\n"
         << "addnode=gridcoin.ddns.net\n"
-        << "addnode=london.grcnode.co.uk\n"
         << "addnode=seeds.gridcoin.ifoggz-network.xyz\n"
         << "addnode=seed.gridcoin.pl\n"
+        << "addnode=swe.tplinkdns.com\n"
```

Comment:
This a dns provider?

Reply:
It's a router manufacturer (Tp-link) that provide ddns service for their own products. That's my node and I have used the address for other stuff for a while without any problems.

Reply:
I'm ok with it as long as its secured to you. My only concern was if u no longer had it and someone took over it while hardcodeds

Reply:
It's secured by my account. Not by the router hardware. Would you suggest that I use a different service?